### PR TITLE
fix(backend):Update with to allow eviction of pods with empty dir. Fixes #4530

### LINKED ISSUE #4530: cluster-autoscaler: safe-to-evict Pod annotation
With the new multi-user pipelines feature, more pods are used which have `emptyDir` (local storage). Local storage causes issues for K8S clusters using the `cluster-autoscaler`, as it will refuse to evict pods which have local storage unless they have the following annotation:
```
cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

We should either make this anotations the default in the manifests, or make this a simple config to change for users.

### PR DESCRIPTION
**Pull request by NikeNano**
**Description of your changes:**
https://github.com/kubeflow/manifests/pull/1752

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

**Comment by Bobgy** [comment]
Hey @NikeNano,

You need to add it in templates/metadata/annotations.
https://www.github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler%2FFAQ.md

It's expected on Pod annotations

**Comment by NikeNano** [comment]
Ohh, my bad ,thanks, will update it. 
> You need to add it in spec.templates.metadata.annotations.
> https://www.github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler%2FFAQ.md
> 
> It's expected on Pod annotations

File: manifests/kustomize/env/gcp/params.env [hunk 1]
```diff
@@ -4,3 +4,4 @@ cacheDb=cachedb
 bucketName=yourGcsBucketName
 gcsProjectId=yourGcsProjectId
 gcsCloudSqlInstanceName=yourCloudSqlInstanceName
+cacheImage='google/cloud-sdk:alpine'
```
Comment (Bobgy) [inline manifests/kustomize/env/gcp/params.env:7]: Nit: Did you take this change from a different branch?

**Review by Bobgy — COMMENTED** [review]
/lgtm

File: manifests/kustomize/env/gcp/params.env (hunk 1 above)
Comment (NikeNano) [reply manifests/kustomize/env/gcp/params.env:7]: good catch, yes i did.

**Comment by Bobgy** [comment]
/lgtm
/approve
Thank you!

**Pull request merged**